client/kif.ts

```diff
@@ -65,17 +65,29 @@ export function resultString(movingPlayerWin: boolean, ply: number, isHandicap:
     }
 }
 
-export function parseKif(text: string) {
-    let date, place, tc, sente, gote, handicap = '';
+export interface KIF{
```

Comment:
There are multiple places where space should be added before/after brackets idk where to start.

Reply:
i think i fixed most i found